devel/libmaxminddb/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        maxmind libmaxminddb 1.3.2
+revision            0
+checksums           rmd160  199373bd1003493fb17ee855b5a5f9ff75a70afa \
+                    sha256  e6f881aa6bd8cfa154a44d965450620df1f714c6dc9dd9971ad98f6e04f6c0f0 \
+                    size    619009
+
+categories          devel net
+platforms           darwin
+license             Apache-2
+maintainers         nomaintainer
+
+description         The libmaxminddb library provides a C library for reading \
+                    MaxMind DB files, including the GeoIP2 databases from MaxMind.
+
+long_description    ${description} This is a custom binary format designed to facilitate \
+                    fast lookups of IP addresses while allowing for great flexibility \
+                    in the type of data associated with an address.\n\n\The \
+                    MaxMind DB format is an open format. The spec is available at \
+                    http://maxmind.github.io/MaxMind-DB/. This spec is licensed under the \
+                    Creative Commons Attribution-ShareAlike 3.0 Unported License.\n\n\See \
+                    http://dev.maxmind.com/ for more details about MaxMind's \
+                    GeoIP2 products.\n
+
+homepage            https://maxmind.github.io/libmaxminddb/
+github.tarball_from releases
+
+depends_build       port:automake port:autoconf port:libtool
+
+variant universal {}
```

Comment:
I don't understand the point of this line. Are you saying this cannot build universal? If so, use this language:

`universal_variant  no
`

Also, there is no point, as far as I am aware, to then add the `get_canonical_archflags` statements below.

Reply:
Ah… ok. That was was prob added when the file was tweaked. It's ok if we remove it.

Then we had a discussion about the maintainer part: (excerpt from the devlist), he wrote:

>I've made a few minor changes: Removed the name line, since github.setup sets it for you; changed the homepage to the one indicated on the GitHub project; and fixed the maintainer line: would you like to maintain the port? If so, I'll list your obfuscated email and "openmaintainer", otherwise "nomaintainer".
>
>I've attached my revised Portfile to this ticket: https://trac.macports.org/ticket/44709

It's ok if you add me as maintainer, if you can arrange that with the obfuscated email part. _Or maybe that part can be taken care of later on?_

Reply:
No, prob not… I removed it, and made the changed  the commit msg the way you suggested.